src/Playlist.vala

```diff
@@ -255,7 +255,7 @@ namespace niki {
             int mediatype;
             bool playnow;
             liststore.get (iter, PlaylistColumns.FILENAME, out file_plying, PlaylistColumns.TITLE, out titlename, PlaylistColumns.FILESIZE, out filesize, PlaylistColumns.MEDIATYPE, out mediatype, PlaylistColumns.ALBUMMUSIC, out album, PlaylistColumns.ARTISTMUSIC, out artist, PlaylistColumns.PLAYNOW, out playnow);
-            NikiApp.settings.set_string ("tittle-playing", titlename);
+            NikiApp.settings.set_string ("title-playing", titlename);
```

Comment:
please change everything related to GLib. Settings
tittle-playing to "title-playing"

Reply:
Already done!

Reply:
topbar.vala
mprisplayer
com.github.torikulhabib.niki.gschema.xml

Reply:
find all related in code "tittle-playing" to "title-playing"